src/python/pants/backend/codegen/protobuf/python/python_protobuf_subsystem.py

```diff
@@ -43,7 +46,6 @@ def register_options(cls, register):
         register(
```

Comment:
Thoughts on moving the above `--mypy-plugin` bool option to the new subsystem? I'm not sure which place make more sense.

Reply:
I think it makes more sense here. Do you want extra feature foo? Great, I'll load a subsystem for that to get the configuration for the extra feature. The use-it-ness is a property of the python-protobuf configuration.

Reply:
That sounds sensible to me. Thanks!